tools/depcheck/pkg/cmd/util/pkg.go

```diff
@@ -108,6 +115,43 @@ func BuildGraph(packages *PackageList, excludes []string) (*depgraph.MutableDire
 	return g, nil
 }
 
+// CalculatePackages receives a set of go import paths and runs "go list"
+// using each path as an entrypoint.
+// Returns a PackageList containing dependency and importPath data for each package.
+func CalculatePackages(entrypoints []string) (*PackageList, error) {
```

Comment:
This looks like it is translate user input into runtime objects.  It belongs close to your command.